kolibri/logger/csv.py

```diff
@@ -47,10 +47,10 @@ def get_content_title(self, obj):
             return ""
 
     def get_time_spent(self, obj):
-        return round(obj.time_spent, 3)
+        return "{:10.1f}".format(round(obj.time_spent, 1))
```

Comment:
Would be more legible. I believe @jamalex wanted seconds to have one decimal point precision. So what would that look like?